plugins/webp-uploads/hooks.php

```diff
@@ -519,65 +519,24 @@ function webp_uploads_remove_sources_files( int $attachment_id ): void {
  * would rely on `wp_filter_content_tags()`.
  *
  * @since 1.0.0
+ * @since n.e.x.t Added the `$context` and `$attachment_id` parameters.
```

Comment:
This really is a new function as it is incompatible with the old function signature. So we should mark it as such, it's more than a change to the existing function.